R/webApiDataSources.R

```diff
@@ -0,0 +1,126 @@
+# @file dataSources
```

Comment:
Rename file to DataSources.R to be consistent with other file names

Reply:
thanks. agreed.